SampleConsoleClient/Program.cs

```diff
@@ -90,7 +92,7 @@ private static void TestGraphQLInterface(string[] args)
 
             };
             var response = graphQLClient.SendQueryAsync<UACloudLibGraphQLObjecttypeQueryResponse>(request).GetAwaiter().GetResult();
-            Console.WriteLine(JsonSerializer.Serialize(response, new JsonSerializerOptions { WriteIndented = true }));
+            Console.WriteLine(System.Text.Json.JsonSerializer.Serialize(response, new JsonSerializerOptions { WriteIndented = true }));

```

Comment:
This isn't using Newtonsoft like you say?

Reply:
The GraphQL code is not using NewtonSoft, only the REST code is.